Dockerfile

```diff
@@ -2,18 +2,15 @@ FROM ubuntu:18.04
 MAINTAINER Lisk <sysops@lightcurve.io>
 ENV DEBIAN_FRONTEND noninteractive
 
-ARG user_name
 ARG user_id
 
-RUN useradd --create-home --uid $user_id $user_name
+RUN useradd --create-home --uid $user_id liskjs
 RUN apt-get update && apt-get install --assume-yes libtool automake autoconf curl python-minimal build-essential git vim || true
 RUN apt-get install --assume-yes sudo postgresql-client-10
-RUN echo "$user_name ALL=(ALL) NOPASSWD: ALL" > /etc/sudoers.d/$user_name
+RUN echo "liskjs ALL=(ALL) NOPASSWD: ALL" > /etc/sudoers.d/liskjs
```

Comment:
What is this used for BTW?

Reply:
in case the user is inside of the container and wants to sudo and install anything else

Reply:
will remove this, it's not necessary